src/Swarm/Game/Step.hs

```diff
@@ -1238,10 +1238,35 @@ execConst c vs s k = do
         inv <- use robotInventory
         return $ Out (VInt (fromIntegral $ countByName name inv)) s k
       _ -> badConst
+    Scout -> case vs of
```

Comment:
So does the `scout` command have an unlimited range?  That seems unrealistic.

Reply:
Fixed